### PR TITLE
Data Remodel

### PR DESCRIPTION
**Pull request by benneely**
This PR contains a remodel based on our discussion (I think). However, I couldn't model everything as discussed. For instance, we talked about making the ImageSet model unique_together with certain colunmns; however, based on the drawing, this was not doable (I commented out what I would have done and added a `todo`). I think it'd be advantageous for you to look at this via a UML diagram and see what changes you'd recommend. I'm really unhappy with the `scrape_imageset_data.py` script. There's a lot of duplication going on and I'm not sure how to fix it. 

In summary, this will work, will keep us in the RDB game, but it just doesn't feel right. Let me know what you think.

### CODE REVIEW

**Comment by whitews** [comment]
Ben, you are right those fields would not be unique together without the concatenated probe/color string we thought we could avoid. So, it looks like we will have to add that field, which in turns brings into question the need for the ImageSetProbeMap table. Let's create a 'probes' field with this string even if it is a bit hackish.

**Comment by whitews** [comment]
Closing after discussion with Ben. We will create a JSONField for 'probes' in the ImageSet table to store the combination of probe names and their colors.
